python_openobserve/openobserve.py

```diff
@@ -25,6 +25,14 @@
     )
     HAVE_MODULE_PANDAS = False
 
+try:
+    import fireducks  # type: ignore
+
+    HAVE_MODULE_FIREDUCKS = True
+except ImportError:
+    print("Can't import fireducks. Some functions may be unavailable.")
```

Comment:
maybe "install this library with name[fireducks] or [all] to support fireducks, otherwise good work ;)